tests/Form/Type/AdminTypeTest.php

```diff
@@ -46,8 +46,8 @@ class AdminTypeTest extends TypeTestCase
 
     protected function setUp(): void
     {
-        $this->adminHelper = $this->createMock(AdminHelper::class);
-        $this->adminType = new AdminType($this->adminHelper);
+        $this->adminHelper = $this->prophesize(AdminHelper::class);
```

Comment:
Why we need to change the mocking implementation?

Reply:
We don't really need to, but all others class/methods was using prophecy in the test of this class. So I made this change for consistency.